main-support.ts

```diff
@@ -231,33 +231,36 @@ export function takeTenScreenshots(
 ) {
 
   if (fs.existsSync(saveLocation + '/' + fileHash + '.jpg')) {
-    //console.log("thumbnails for " + fileHash + " already exist");
+    // console.log("thumbnails for " + fileHash + " already exist");
     done();
   }
 
   let current: number = 1;
   const totalCount = 11;
   const step: number = duration / totalCount;
-  var args = [];
-  let concat = "";
+  const args = [];
+  let concat = '';
 
   // make the magic filter
   while (current < totalCount) {
-    let time = current * step;
+    const time = current * step;
     args.push('-ss', time, '-i', pathToVideo);
-    concat += "[" + (current - 1) + ":v]";
+    concat += '[' + (current - 1) + ':v]';
     current++;
   }
-  args.push('-frames', 1, '-filter_complex', concat + "vstack=inputs=" + (totalCount - 1), saveLocation + '/' + fileHash + '.jpg');
-
-  const ffmpeg = spawn(ffmpegPath, args);
-  ffmpeg.stdout.on('data', function (data) {
-    console.log(data);
-  });
-  ffmpeg.stderr.on('data', function (data) {
-    console.log('grep stderr: ' + data);
-  });
-  ffmpeg.on('exit', () => {
+  args.push('-frames', 1, '-filter_complex',
```

Comment:
Can you move `-filter_complex` to the line below, so the flag and params are together?

Reply:
Hope this is what you meant -- please see the below commit